lib/datadog/tracing/contrib/graphql/ext.rb

```diff
@@ -7,11 +7,11 @@ module GraphQL
         # GraphQL integration constants
         # @public_api Changing resource names, tag names, or environment variables creates breaking changes.
         module Ext
```

Comment:
These `Ext` changes were done for all integrations, except GraphQL in https://github.com/DataDog/dd-trace-rb/pull/1800. Because GraphQL changes involved upstream changes, we held off until now to make them.

Reply:
Hmmm, that makes sense.
Nothing we can do in this PR, unfortunately.
I'll take note of this so we either quickly release a new version of ddtrace or update upstream.